build.gradle

```diff
@@ -104,7 +104,7 @@ if (!project.hasProperty('bytemanVersion')) {
   ext.bytemanVersion = '2.2.1'
 }
 
-ext.avroVersion = '1.7.7'
+ext.avroVersion = '1.8.0'
```

Comment:
Has version `1.8.0` been tested? Release notes say it may contain incompatible changes: https://issues.apache.org/jira/browse/AVRO?selectedTab=com.atlassian.jira.jira-projects-plugin:changelog-panel